### ISSUE TITLE
kiosk database questions

### ISSUE THREAD

**Issue by eileenpeppard** [erdl/survey_admin#20]
Hey, Carlos. i don't know where to put this issue. I'm looking at the kiosk schema and I think there should be a link between kiosk_survey and other tables but I don't know how.  also, not shown in this screenshot: there is one more table called "user" that I can't even access. Is that table used? If not, can you delete it?
![schema](https://user-images.githubusercontent.com/29083044/33461987-1160b9d2-d5d9-11e7-8d8c-ea729d8182be.PNG)

**Comment by carlosparadis** [comment]
blocked by #6 and another issue. @ryantanaka what database project issue links to this? Or any other about the user users part.

**Comment by ryantanaka** [comment]
@carlosparadis ,

This was the issue from survey_display (https://github.com/erdl/survey_display/issues/15)

**Comment by carlosparadis** [comment]
#47 is a continuation to this as it replaces the existing user table by Oauth. So I am closing this one as it is redundant.
